example/android/app/build.gradle

```diff
@@ -28,12 +28,12 @@ apply plugin: "kotlin-android"
 apply from: "$flutterRoot/packages/flutter_tools/gradle/flutter.gradle"
 
 android {
-    compileSdkVersion(30)
+    compileSdkVersion(31)
     buildToolsVersion = "30.0.3"
```

Comment:
Since you change the compile sdk version you should set buildToolsVersion to "31.0.0"

Reply:
This raises an error "Installed Build Tools revision 31.0.0 is corrupted". As Build tools version 30.0.3 works well for running the sample, I don't think we need to change it.

Reply:
Did you try to reinstall the build tools?
This is very strange. Maybe you should revert api 30 while we investigate the issue with build tools 31.0.0.
We should use the same api for compile sdk and build tools

Reply:
It seems the build-tools themselves are corrupted, not sure why: https://stackoverflow.com/questions/68387270/android-studio-error-installed-build-tools-revision-31-0-0-is-corrupted
Reverting API 30 will trigger a build error because of some Flutter dependency.
As this is just the build file for the sample and not the distributed plugin, I would not worry about it too much.

Reply:
ok

Reply:
I added a comment to explain the situation